scientific/src/commonMain/kotlin/converter/decimal/convertFromDimensionless.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.converter.decimal
+
+import com.splendo.kaluga.base.utils.Decimal
+import com.splendo.kaluga.base.utils.toDecimal
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import com.splendo.kaluga.scientific.ScientificValue
+import com.splendo.kaluga.scientific.div
+import com.splendo.kaluga.scientific.invoke
+import com.splendo.kaluga.scientific.times
+import com.splendo.kaluga.scientific.unit.One
+
+infix operator fun Decimal.div(modifier: ScientificValue<PhysicalQuantity.Dimensionless, *>): Decimal {
+    val scalar = this.invoke(One)
+    val result = scalar / modifier
+    return result.value.toDecimal()
+}
+
+infix operator fun Decimal.times(modifier: ScientificValue<PhysicalQuantity.Dimensionless, *>): Decimal {
```

Comment:
You should also support the inverse here

Reply:
Hmm... what should be expected result of: 5(Percent) * 5.0 ?
- 0.25(Decimal)
- 25(Percent)

Reply:
same for division: 5(Percent) / 5.0 ?
- 0,01(Decimal)
- 1(Percent)

Reply:
Id say its the same result at 5(Percent) / 5(One)

Reply:
I think we can assume that while mathematically the same following operations are not semantically equal:
1. 5(Percent) * 5.0(Decimal)
2. 5(Decimal) * 5.0(Percent)

Hence I propose this solution:
 - `5(Decimal) * 5.0(Percent) = 0.25(Decimal)`
 - `5(Percent) * 5.0(Decimal) = 0.25(One)`